sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/SearchOptions.java

```diff
@@ -95,6 +95,22 @@ public final class SearchOptions implements JsonSerializable<SearchOptions> {
      */
     private String semanticConfiguration;
 
+    /*
+     * Allows the user to choose whether a semantic call should fail completely, or to return partial results.
+     */
+    private SemanticErrorHandling semanticErrorHandling;
```

Comment:
The updates to this class will need to be copied into the `SearchOptions` in the public API. There will also need to be a few additional properties added to handle `queryAnswerThreshold`, similar to what was done for `queryAnswerCount`.